packages/node_modules/@webex/plugin-meetings/test/unit/spec/meetings/index.js

```diff
@@ -293,6 +312,18 @@ skipInBrowser(describe)('plugin-meetings', () => {
       });
     });
     describe('Private Detailed API and Helpers', () => {
+      describe('#listenForEvents', () => {
+        beforeEach(() => {
+          webex.meetings.handleLocusMercury = sinon.stub().returns(true);
+          webex.internal.mercury.on = sinon.stub().returns((type, callback) => {
+            callback();
+          });
+        });
+        it('Should register for mercury events', () => {
+          webex.meetings.listenForEvents();
+          assert.calledThrice(webex.internal.mercury.on);
```

Comment:
Can this test be updated to be a bit more specific and use the `assert.calledWith` for each thing it is supposed to be listening to?

Reply:
done